public_html/includes/js/develop/createForm_form.js

```diff
@@ -1531,8 +1531,9 @@ function addMetadataStandard(val){
 	}
 
 	var options    = metadataSchema;
-	var identifier = val.split(' : ')[1];
-	var standard   = val.split(' : ')[0];
+	val = val.replace(/\s+/g, '');
```

Comment:
is identifier or standard used any other places, such as the php, or being inserted into the forms before it is posted?